### PR TITLE
Upgrated Kurtosis dependencies

### PR DESCRIPTION
**Pull request by leoporoli**

### CODE REVIEW

**Review by mieubrisse — APPROVED** [review]

File: docs/changelog.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # TBD
+### Changes
+* Upgrade to core to 1.58.1 and engine to 1.32.1
```
Comment (mieubrisse) [inline docs/changelog.md:3]: Is this a breaking change, or is it okay?

Comment (mieubrisse) [inline docs/changelog.md:3]: Is this a breaking change, or is it okay?

Comment (leoporoli) [reply docs/changelog.md:3]: it is not a breaking change because users could use this module version without changing anything

Comment (leoporoli) [reply docs/changelog.md:3]: it is not a breaking change because users could use this module version without changing anything

**Pull request merged**